### ISSUE TITLE
Tests shouldn't fail when season rolls over [40,13,3.1]

### ISSUE THREAD

**Issue by stenington** [Iridescent-CM/technovation-app#2550]
Last year, certain tests began failing when the season rolled over until date config was updated, and certificate templates were copied into a new directory. See https://github.com/Iridescent-CM/technovation-app/pull/2233 

I don't think the test suite should pass one day and fail the next, with no changes in between other than the actual date. These tests should probably be abstracted away from the actual app settings, and maybe we should be testing multiple cases (e.g. before a configured date, after a configured date).

**Comment by stenington** [comment]
These are the failures I get by setting the 2020 dates in my `.env` back to 2019, so it doesn't include failures for missing certificate templates:

```
rspec ./spec/features/mentor/find_a_mentor_spec.rb:36 # Mentors find a team browse nearby mentors
rspec ./spec/features/mentor/find_a_mentor_spec.rb:117 # Mentors find a team visit the mentor page
rspec ./spec/features/mentor/find_a_team_spec.rb:114 # Mentors find a team request to join a team
rspec ./spec/features/mentor/find_a_team_spec.rb:132 # Mentors find a team request to join the same team again, even if you deleted an earlier request
rspec ./spec/features/student/find_a_team_spec.rb:31 # Students find a team request to join a team
rspec ./spec/features/student/find_a_team_spec.rb:45 # Students find a team onboarded student sees pending requests
rspec ./spec/system/student/request_to_join_a_team_spec.rb:79 # Students request to join a team a valid student requestor student accepts the request
rspec ./spec/system/student/request_to_join_a_team_spec.rb:118 # Students request to join a team a valid student requestor student accepts from team page
rspec ./spec/system/student/request_to_join_a_team_spec.rb:132 # Students request to join a team a valid student requestor student declines from team page
rspec ./spec/system/student/request_to_join_a_team_spec.rb:146 # Students request to join a team a valid student requestor mentor accepts the request
rspec ./spec/system/student/request_to_join_a_team_spec.rb:165 # Students request to join a team a valid student requestor mentor accepts from team page
rspec ./spec/system/student/request_to_join_a_team_spec.rb:179 # Students request to join a team a valid student requestor mentor declines from team page
rspec ./spec/system/student/request_to_join_a_team_spec.rb:193 # Students request to join a team a valid student requestor student declines the request
rspec ./spec/system/student/request_to_join_a_team_spec.rb:219 # Students request to join a team a valid student requestor mentor declines the request
```

**Comment by stenington** [comment]
Rather than setting 2019 variables and letting it be the 2020 season to trigger failures, I tried setting 2020 config but making the season rollover early enough that it's the 2021 season. This seems to have triggered failures in calculating divisions on top of the failures listed above.

Also the extended capybara timeouts that we've had to put in place for certain pages using Vue are really painful when tests start breaking. I'm not sure what to do about it, but it's another downside of how we're currently working with Vue, I think.
